src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -133,4 +134,220 @@ private ResultActions getResultActions(HttpMethod httpMethod, String path,
                 .content(asJsonString(dto));
         return mockMvc.perform(builder).andExpect(matcher);
     }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    void getChild_Success() throws Exception {
+        mockMvc.perform(get("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$", hasSize(2)))
+                .andExpect(jsonPath("$[0].id").value(1L))
+                .andExpect(jsonPath("$[0].sex").value("male"))
+                .andExpect(jsonPath("$[1].id").value(2L))
+                .andExpect(jsonPath("$[1].sex").value("female"));
+
+    }
+
+    @Test
+    @WithMockUser(username = "no-such-user")
+    @Transactional
```

Comment:
Why did you use Transactional here if the user is not exist and the database wouldn't be invoked?